keyboard.html

```diff
@@ -126,19 +126,19 @@ <h2 id="film" class="no-display">Video on Keyboard Compatibility</h2>
       <p>[ <strong>Previous:</strong> <em>none</em> | <a href="Overview.html">Up: Overview</a> | <a href="contrast.html"><strong>Next:</strong> Colors with Good Contrast</a> ]</p>
 
       <h2 id="what">What is Keyboard Compatibility?</h2>
-      <p>All functionality needs to work with the keyboard. That is, users can access and move between links, buttons, forms, and other controls using the Tab key and other shortcuts. Websites should not require a mouse; for example, pop-up calendars should also let users type in a date.</p>
+      <p>All functionality needs to work with the keyboard. That is, users can access and move between links, buttons, forms, and other controls using the Tab key and other keystrokes. Websites should not require a mouse; for example, pop-up calendars should also let users type in a date.</p>
 
       <h2 id="who">Who depends on this feature?</h2>
       <ul>
         <li>People with physical disabilities who cannot use the mouse.</li>
         <li>People who are blind, and cannot see the mouse pointer on the screen.</li>
-        <li>People with chronic conditions, such as repetitive stress injuries (RSI), who should not use a mouse much.</li>
+        <li>People with chronic conditions, such as repetitive stress injuries (RSI), who should limit or avoid use of a mouse.</li>
       </ul>
 
       <h2 id="others">What are the additional benefits?</h2>
       <ul>
-        <li>Content works for people with temporary limitations, such as a broken arm or broken mouse.</li>
-        <li>Content works in more situations, such as with on-screen keyboards and public terminals without a mouse.</li>
+        <li>Keyboard interface works for people with temporary limitations, such as a broken arm or broken mouse.</li>
+        <li>Interface works in more situations, such as with on-screen keyboards and public terminals without a mouse.</li>
```

Comment:
Things with onscreen keyboards, or public terminals without a mouse, are not an obviously realistic example. 

Onscreen keyboards work great with a mouse - or with a touch interface, but that isn't what is being discussed here. I suggest removing the example.